coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/conflation/GlobalBlobAwareConflationCalculatorTest.kt

```diff
@@ -43,6 +52,17 @@ class GlobalBlobAwareConflationCalculatorTest {
   private lateinit var blobs: MutableList<Blob>
   private val defaultBatchesLimit = 2U
   private val conflationTargetEndBlockNumbers: MutableSet<ULong> = mutableSetOf()
+  private lateinit var mockMetricsFacade: MetricsFacade
+
+  // histogram metrics mocks
+  private lateinit var mockGasUsedInBlobHistogram: Histogram
```

Comment:
I feel like this is an ideal case for a fake implementation backed by a list. Mocks instantiation is too repetitive IMO and the only advantage we're taking from the mocks is the `inOrder` verification. IMO a simple Histogram implementation will cover our needs much better with less repetitive and more readable code